### ISSUE TITLE
Flashstart fails to send heartbeats

### ISSUE THREAD

**Issue by Tbaile** [NethServer/nethsecurity#955]
**Steps to reproduce**

- Activate Flashstart filter

**Expected behavior**

- Service activates at first, then pings every 30 minutes

**Actual behavior**

- Service activates, but then no other ping is made

**Components**

Image version: 8-23.05.5-ns.1.3.0

**See also**

- [Mattermost discussion](https://mattermost.nethesis.it/nethesis/pl/95jdxi796prj5kuxd6urb7qfyr)

**Comment by gsanchietti** [comment]
Testing image: 23.05.5-ns.1.3.0-93-g225c40bf


**Test case**
Check the issue is not reproducible

**Comment by stephdl** [comment]
QA

- used the image nethsecurity-8-23.05.5-ns.1.3.0-104-gaf4379a6
- Enable flashstart with credentials
- waiting 30 minutes
- find this in logs
```
Dec 10 10:30:00 NethSec crond[3906]: USER root pid 8707 cmd sleep $(( RANDOM % 60 )); /usr/share/ns-flashstart/flashstart-auth
Dec 10 10:30:06 NethSec nethsecurity-api[5157]: nethsecurity_api 2024/12/10 10:30:06 middleware.go:169: [INFO][AUTH] authorization success for user root. POST /api/ubus/call {"path":"ns.flashstart","method":"set-config","payload":{"enabled":true,"username":"user@domain.com","pa
ssword":"XXX","zones":["lan"],"bypass":[]}}
Dec 10 10:30:06 NethSec nethsecurity-api[5157]: nethsecurity_api 2024/12/10 10:30:06 middleware.go:169: [INFO][AUTH] authorization success for user root. GET /api/refresh 
Dec 10 10:30:06 NethSec nethsecurity-api[5157]: nethsecurity_api 2024/12/10 10:30:06 middleware.go:199: [INFO][AUTH] refresh response success for user root
Dec 10 10:30:06 NethSec nginx: 192.168.61.1 - - [10/Dec/2024:10:30:06 +0000] "GET /api/refresh HTTP/1.1" 200 252 "https://192.168.61.134/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
Dec 10 10:30:07 NethSec netdata[7267]: Initializing file /var/cache/netdata/disk_qops.sda/main.db.
Dec 10 10:30:07 NethSec netdata[7267]: Initializing file /var/cache/netdata/disk_qops.sda/operations.db.
Dec 10 10:30:07 NethSec dnsmasq[1]: exiting on receipt of SIGTERM
Dec 10 10:30:07 NethSec nginx: 192.168.61.1 - - [10/Dec/2024:10:30:07 +0000] "POST /api/ubus/call HTTP/1.1" 200 83 "https://192.168.61.134/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
Dec 10 10:30:07 NethSec dnsmasq[1]: started, version 2.90 cachesize 1000
Dec 10 10:30:07 NethSec dnsmasq[1]: DNS service limited to local subnets
Dec 10 10:30:07 NethSec dnsmasq[1]: compile time options: IPv6 GNU-getopt no-DBus UBus no-i18n no-IDN DHCP DHCPv6 no-Lua TFTP conntrack ipset nftset auth cryptohash DNSSEC no-ID loop-detect inotify dumpfile
Dec 10 10:30:07 NethSec dnsmasq[1]: UBus support enabled: connected to system bus
Dec 10 10:30:07 NethSec dnsmasq[1]: using nameserver 127.0.0.1#5300
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for test
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for onion
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for localhost
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for local
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for invalid
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for bind
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for lan
Dec 10 10:30:07 NethSec dnsmasq[1]: reading /tmp/resolv.conf.d/resolv.conf.auto
Dec 10 10:30:07 NethSec dnsmasq[1]: using nameserver 127.0.0.1#5300
Dec 10 10:30:07 NethSec dnsmasq[1]: using nameserver 192.168.61.1#53
Dec 10 10:30:07 NethSec dnsmasq[1]: using nameserver 192.168.100.1#53
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for test
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for onion
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for localhost
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for local
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for invalid
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for bind
Dec 10 10:30:07 NethSec dnsmasq[1]: using only locally-known addresses for lan
Dec 10 10:30:07 NethSec dnsmasq[1]: read /etc/hosts - 12 names
Dec 10 10:30:07 NethSec dnsmasq[1]: read /tmp/hosts/dhcp.cfg01411c - 0 names
```

the cron is executed alone, set verified

**Comment by gsanchietti** [comment]
Released on 23.05.5-ns.1.4.0
